src/components/TalkToUs/style.js

```diff
@@ -31,9 +13,19 @@ export const helpButton = theme => css`
   text-align: center;
   color: ${theme.colors.purples.normal};
   padding: 1.5rem 2rem;
+  position: fixed;
+  right: 1.5rem;
+  bottom: 1.5rem;
+  z-index: ${theme.zIndices.content};
+  box-shadow: 0 0 1rem #000000;
+  outline: none;
+
+  ${opening && `transition: bottom .2s ease-out; bottom: -6rem;`}
+  ${closing &&
+    `transition: bottom .2s ease-out; transition-delay: .3s; bottom: 1.5rem;`}
```

Comment:
transition related css can be simplified as
```
bottom: ${opening ? "-6rem" : "1.5rem"};
transition: bottom 0.2s ease-out;
${!opening && "transition-delay: .3s"};
```

Reply:
fixed.